theseus_gui/src/pages/Settings.vue

```diff
@@ -177,6 +179,10 @@ async function refreshDir() {
           Refresh
         </Button>
       </div>
+      <label for="theme">
+        <span class="label__title">App version</span>
+        <span class="label__description"> Your app has the version {{ launcherVersion }} </span>
```

Comment:
could we change the wording of this and make it fit in more with the UI? i feel that a simple `Theseus v{{ launcherVersion }}` in a different area would do.

Reply:
you were suggesting this:
![grafik](https://github.com/modrinth/theseus/assets/81473300/6a1c86e4-fa1e-453a-aedd-1b7c84d7c199)

right?

Reply:
yes, this fits better